script.module.inputstreamhelper/addon.xml

```diff
@@ -1,25 +1,42 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<addon id="script.module.inputstreamhelper"
-       version="0.3.5"
-       name="InputStream Helper"
-       provider-name="emilsvennesson">
+<addon id="script.module.inputstreamhelper" version="0.4.1" name="InputStream Helper" provider-name="emilsvennesson,dagwieers">
     <requires>
         <import addon="xbmc.python" version="2.25.0"/>
+        <!-- This is needed for the plugin to import the library -->
+        <import addon="script.module.inputstreamhelper" optional="true"/>
         <import addon="script.module.kodi-six" version="0.0.4"/>
-        <import addon="script.module.requests" version="2.9.1"/>
+        <import addon="script.module.pysocks" version="1.6.8" optional="true"/>
     </requires>
-    <extension library="lib" point="xbmc.python.module"/>
+    <extension point="xbmc.python.module" library="lib"/>
+    <!-- This is needed to get an addon icon -->
+    <extension point="xbmc.python.script" library="addon.py"/>
     <extension point="xbmc.addon.metadata">
         <description lang="en_GB">A simple Kodi module that makes life easier for add-on developers relying on InputStream based add-ons and DRM playback.</description>
         <description lang="el_GR">Μία απλή μονάδα για το Kodi η οποία διευκολύνει την ζωή των προγραμματιστών οι οποίοι εξαρτώνται από τα πρόσθετσ InputStream και αναπαραγωγή τύπου DRM.</description>
-        <news>2019.08.15 v0.3.5
-        + Auto install inputstream.adaptive (mediaminister)
-        + Fix latest Widevine version detection (dagwieers)
-        + Check for Widevine updates on new release (mediaminister)
-    </news>
+        <description lang="ru_RU">Простой модуль для Kodi, который облегчает жизнь разработчикам дополнений, с использованием InputStream дополнений и воспроизведения DRM контента.</description>
+        <news>
+v0.4.1 (2019-09-01)
+- Use local url variable
+- Directly use Kodi CDM directory
+- Implement settings menu and API
+- Add integration tests
+- Add a progress dialog for extraction on ARM
+- Fix crash when using platform.system()
+- Fix a python error
+- Remove legacy Widevine CDM support
+- Replace requests/urllib3 with urllib/urllib2
+- Various unicode fixes
+- Add proxy support
+- Add setting to disable inputstreamhelper
+- Check Widevine support before all checks
+- Support 64-bit kernel with 32-bit userspace
+- Dutch, German, Greek, Italian, Russian and Swedish translations
+        </news>
         <platform>all</platform>
         <license>MIT License</license>
         <source>https://github.com/emilsvennesson/script.module.inputstreamhelper</source>
+        <website>https://github.com/emilsvennesson/script.module.inputstreamhelper/wiki</website>
+        <forum>https://github.com/emilsvennesson/script.module.inputstreamhelper/issues</forum>
```

Comment:
I think this is only in case you have a forum thread in Kodi forums. If there is no support there, and since the website element already points to the same location, consider to remove the element